### ISSUE TITLE
Feedback from XMage Cersion 1.4.30V6

### ISSUE THREAD

**Issue by LevelX2** [magefree/mage#5126]
# 1.4.30V6
--
- [ ] Skyrider Patrol: the effekt stacks cumulative. Every turn you use it you get to put one more counters (reported by Termaran / 2018-06-25 13:07:24.695).
- [ ] BUG: Skyrider patrol can multiple triggers at start on combat, its like +1 on each turn its on board (reported by hexilon21 / 2018-06-25 16:47:08.461).
- [ ] Rise // Fall bugged: Rise//Fall caused the game to freeze. Thank you for all your work! <3 (reported by jdragsky / 2018-06-25 20:21:35.672).
- [ ] Delete his account and ban him: In my last game a Jund-Player took 3 turns in a row at the beginning until I was dead. I dont know his name,only that it started with "J".I was so pissed, I directly left the game (reported by ChaosKai / 2018-06-26 00:47:49.101).
- [ ] Kicking people: Who ever made the game should be able to kick people if somebody disconnects or something (reported by Cpudemon / 2018-06-26 02:23:41.664).
- [ ] Tezzeret, Cruel Machinist's Ultimate Broken: Tezzeret, Cruel Machinist's ultimate is reportedly broken. Thank you so much for getting M19 into the game so quickly! (reported by cOreGon / 2018-06-26 04:12:24.268).
- [ ] New Ajania PW ult does not work PLEASE FIX: the new white ajania planes walker ult does not work. after i made the emblem it would just skip over my end step without making the cat tokens please fix (reported by Gonegold / 2018-06-26 05:43:00.23).
- [ ] game emptied/started mid-game: the game stopped midgame, calling "your game has started", apparently setting both players to an empty game (reported by kazmcfo / 2018-06-26 12:16:17.43).
- [ ] Pollenbright wings bug: Pollenbright wings untaps all his controller lands after attacking (reported by Colnakdc / 2018-06-26 13:58:58.049).
- [ ] Swap the Tiny Leader sort option for Pauper: I never see games for Tiny Leader up, but even at 4am (pst) there are always a couple Pauper games running. It's a little thing that would make sorting a touch easier.Also you're so cool devs! Good job so far! (reported by CamelWizard / 2018-06-26 14:30:34.647).
- [ ] card malfunction: departed deckhand (M19) does have power and toughness (2/2)but the card apears to be 0/0, and does not stick to the bf (reported by MageBR / 2018-06-26 14:32:53.316).
- [ ] drag drop glitch: dragging and dropping cards after a draft on mac osx sometimes causes the cards to disappear forever (reported by Krebsy / 2018-06-26 19:27:02.85).
- [ ] Bring to light and Scapeshift: Player cast Bring To Light, searching for Scapeshift and casting it, opp cast Remand on Scapeshift. After that, player A can´t cast Scapeshift, it seems like an abbility that doesn´t work (reported by plaguba / 2018-06-27 03:41:39.134).
- [ ] PLEASE FIX TEFERI, HERO OF DOMINARIA BUG: When +1 ability triggers at the end step, if I have less than 2 lands to untap the done button does not appear and wont allow me to skip through the ability or stack. HELP! (reported by blueXmage / 2018-06-27 05:41:52.074).
- [ ] speed up the servers!: Servers lag too much! (reported by Lukejr / 2018-06-27 11:37:22.556).
- [ ] Jaya Emblem: Jaya emblem is returning spells to the graveyard (reported by FrankDammit / 2018-06-27 12:31:01.641).
- [ ] Skyrider Patrol bug (M19 card): the trigger that happens before combat can be used multiple times per turn? (reported by james5265 / 2018-06-27 12:56:05.642).
- [ ] Rowen: Ult does nothing (reported by FrankDammit / 2018-06-27 13:09:58.816).
- [ ] Reality Spasm: when selecting targets the spell waits until resolution to have you pick targets, when tapping at least. I don't know about untapping. (reported by FrankDammit / 2018-06-27 13:11:04.46).
- [ ] BUG Consign/Oblivion card: and enemy cast Consign/Oblivion targeting my crit. i'm pressing done, because i have no answer for that and then i can't move on. Itl like freeze but it's still my turn, my time and none of F keys (f2f7,f6) seems to work for me (reported by Dansoni / 2018-06-27 18:22:50.92).
- [ ] skyrider patrol: sometiems gives 2 coutners (reported by Will / 2018-06-27 19:56:46.256).
- [ ] skyrider patrol: should only trigger once, not once and then once again for each +1/+1 counter on a creature you controlthank you for your hard work (reported by Will / 2018-06-27 20:00:24.657).
- [ ] Fix Blood Moon: Lands that normally enter tapped should not enter tapped with Blood Moon in play, but they do. (reported by Haruchan / 2018-06-27 20:53:30.53).
- [ ] Bugs: the servers seem to lag or bug out ALOTits very frustrating (reported by Khornedogg / 2018-06-27 23:53:43.993).
- [ ] highland lake m19: should enter battlefield tapped (reported by Will / 2018-06-28 02:55:41.924).
- [ ] the eldest reborn bugged: Chapter I allows the opponent to choose creatures they don't control to sacrifice (reported by Haywire / 2018-06-28 10:56:39.32).
- [ ] Vivien Reid: second ability should be -3 not -X (reported by Will / 2018-06-28 17:23:28.072).
- [ ] sift m19: should be "draw 3 cards then discard 1 card" not "draw 3 cards" (reported by Will / 2018-06-28 18:38:28.865).
- [ ] BRINE SHAMAN: Mouse-over text says non-creature, card says -summon. This is in conflict. (reported by RedSheBathed / 2018-06-29 19:14:08.115).
- [ ] brine shaman: Brine shamans card says counter target summon spellHowever the mouseover text says counter target noncreature spell (reported by ToppedDeck / 2018-06-29 19:14:29.892).
- [ ] blood moon: please make blood moon work correctly (reported by Lukas96 / 2018-06-30 00:57:34.427).
- [ ] Skyrider Patrol: Skyrider Patrol triggers twice the second time its beinbg used (reported by DonKarnage / 2018-06-30 01:26:57.618).
- [ ] Arcades, the Strategist: allows all creatures with defender to attack, not just creatures you controlthank you for hard work on client (reported by Will / 2018-06-30 01:39:57.804).
- [ ] Ajani From m19 emblem: using Ajani emblem doesnt work. cats are not generated at end step (reported by Zox / 2018-06-30 06:48:09.424).
- [ ] Skyrider stack bug: skyrider Patrol is bugged. Everyturn you get one +1/+1 counter additional (reported by Termaran / 2018-06-30 10:55:32.187).
- [ ] Disrupt Decorum: Goad doesnt work with only one opponent. It doesnt allow to move past declare attacks without attacking, but also forbids attacking (reported by Bob49 / 2018-06-30 11:41:02.237).
- [ ] Resoolution is so tiny I can't use: Resoolution is so tiny I can't use (reported by smille63 / 2018-06-30 22:54:21.939).
- [ ] Terzzeret's ultimate doesn't work doesn't work: yep, almost killed myself (reported by drowinternet / 2018-07-01 00:58:42.707).
- [ ] concede glitch: you guys really need to fix that glitch where someone leaves a multiplayer game without conceding and their permanents stick around. it ruins alot of edh games. (reported by igniteaxiom / 2018-07-01 03:43:01.999).
- [ ] Equinox Missing: I noticed that the card "Equinox" from the "Legends" set is not included in the card directory. Is there a reason for this? Thank you! (reported by BraveSirRobn / 2018-07-01 07:17:16.244).
- [ ] Shown wording of Gauntlet of Power is wrong: The shown wording of gauntlet of power doesn't match the oracle one. Seems to work fine though (reported by DerLhurgoyf / 2018-07-01 08:54:38.007).
- [ ] Skyrider Patrol doesn't clear in between turns: Skyrider Patrols ability lets you put an additional counter for each time you've already activated its ability this game. Probably just not cleared (reported by Haywire / 2018-07-01 09:19:29.776).
- [ ] Fix a couple of broken emblems: The emblems for Rowan Kenrith and Tezzeret, Artifice Master currently do absolutely nothing and just sit there. Tezzeret doesn't search permanents on end step and Rowan doesn't copy abilities that she should be able to, like planeswalker abilities. (reported by Lightslayer247 / 2018-07-01 12:51:02.421).
- [ ] Skyrider patrol Bug: when you pay for the trigger it resolves the total times you have payed instead of one per turn (reported by JoElVago / 2018-07-01 23:38:39.784).
- [ ] Bug with poison tip archer: Poison tip archer does not trigger on my own creatures. (reported by Azeraph / 2018-07-02 05:29:24.423).
- [ ] Ajani, Adversary of Tyrants: Ultimated Ajani and got the emblem. Never got any Cats on end step. (reported by ServerTimeout / 2018-07-02 05:32:05.342).
- [ ] cant play: all of the sudden this msg keeps popping " server erros card not found" if i take that card out the error goes to another card (reported by poisoN / 2018-07-02 15:51:08.094).
- [ ] cards don't show up on tournaments: cards don't show up on tournaments.Thank you ;) (reported by osebius / 2018-07-02 17:14:43.794).
- [ ] Server Bug: All playeres are Unable to join or create game. Error message that cards used are diferent version of the server.Thx. (reported by FirstBornEvil / 2018-07-02 22:42:34.825).
- [ ] Server not working: nothing seems to be working (reported by bitbull / 2018-07-03 02:51:06.888).
- [ ] Ajani, Adversary of Tyrants: -7 ability does not workthank you for edits (reported by Will / 2018-07-04 18:21:36.005).
- [ ] common lands in m19: do not enter battlefield tappedthank you for work (reported by Will / 2018-07-04 18:33:51.574).
- [ ] Growing Rites of Itlimoc: When I select this card, it will give me an option to either cast or cancel. When I click on cast, the options disapears and nonthing happens. This happens no matter how much mana is in my pool and how many land sI have untapped (reported by shaqattack / 2018-07-05 05:53:59.557).
- [ ] Vivien of the Arkbow Ability Cost: The cost related to Vivien of the Arkbow's second ability is incorrect. Currently, it is set to -X, and you select a cost. However, it should be -3. (reported by jfine / 2018-07-05 14:21:58.689).
- [ ] skyrider patrol: to put a the card is bugged, after i pay the mana cost it let's me choose creatures to put +1/+1 counters on.But i can choose more than one creature (reported by DutchEagle / 2018-07-05 16:10:16.133).
- [ ] councils judgement: Add the identity[xyz] in the text, so that i know what the oponent choose.If i have multiple cards with the same name this gets realy wierd (reported by badmoon / 2018-07-06 11:03:26.703).
- [ ] skyrider patrol: Skyrider patrol in M19 doesnt work properly. Counters triggers multiple times and you cant do anything to prevent his (reported by _Ezekiel_ / 2018-07-06 19:40:19.263).
- [ ] vivien reid -3: the -3 is giving me the option to -x instead (reported by Karamel / 2018-07-06 22:57:45.065).
- [ ] Images: When downloading images, you have for all standard. I would like one for all modern. (reported by akuma508 / 2018-07-06 23:16:54.939).
- [ ] Triggers duplicating on the stack: When I pay UG for the trigger on Skyrider Patrol (M19), if I spam click the target creature I control, I can make the trigger happen multiple times. (reported by Shibo / 2018-07-07 02:08:45.223).
- [ ] Archon of Valor's Reach Bug report: On the card "Archon of Valor's Reach", when the card type "Planeswalker" was chosen, the effect did not stop the opponent from playing a planeswalker. (reported by MichelinMan5 / 2018-07-07 06:53:19.983).
- [ ] Lava Burst (IA) is missing: not in the database?! (reported by HAL9000 / 2018-07-07 15:36:10.569).
- [ ] Call of the Kindred bug: The Card "Call to the Kindred" didn't work. If you chose "yes" it is skipping the turn. (reported by JackSchubi / 2018-07-07 20:21:41.88).
- [ ] Heart Piercer Sacrifice: I embalmed a heart-piercer manticore, chose to sacrifice another unembalmed heart-piercer manticore to it's ETB ability, and the sacrifice occurred, but no damage was dealt (reported by Haywire / 2018-07-07 21:25:38.126).
- [ ] Ability to kick people from table: Some people here are too cancerous and i dont want them joining my tables and drafts, allow us to kick them (reported by OathOfRhino / 2018-07-07 21:45:53.362).
- [ ] no images: images not download still missing 1300 images (reported by argon / 2018-07-08 01:16:39.398).
- [ ] Bug: 'Order of Succession': it does nothing sadly enough, I cast it, get a prompt to choose elft or right and then it goes to the graveyard without resolving anything, (reported by Telandril / 2018-07-08 01:42:23.998).
- [ ] Ajani, Adversery of Tyrants ult doesn't work: Emblem goes into command zone, ajani dies at 7, but no cats and end step. Kind of a big deal with an ult being super bugged like this. Especailly for standard playing. Not to mention commander. (reported by Quickster203 / 2018-07-08 03:31:26.222).
- [ ] Vivien Reid: Vivien Reids loyatly counters don't remove on the second option. should be minus 3, not x (reported by TestofAdam / 2018-07-08 07:51:14.012).

**Comment by LevelX2** [comment]
- [ ] Vivien reid - is wrong: Vivein reid's - ability is set to -x instead of -3 like it should be. please fix (reported by ovenmitt / 2018-07-08 13:14:40.498).
--
- [ ] Skyrider Patrol: It's at the beggining of combat on your turn. It should be one trigger but it is able to be payed multiple times (reported by Czarshark / 2018-07-08 14:49:36.96).
- [ ] one with the machine: I drew 5 cards with highest cmc being 2 (reported by Karamel / 2018-07-08 15:13:55.889).
- [ ] I cannot dowload m19 images (theyr not a thing9: (300 characters max) (reported by BoarWithNoSoul / 2018-07-09 00:44:37.008).
- [ ] blood moon with shock lands: blood moon makes it so shock lands should come in untapped without having to pay 2 life (reported by EplyMTG / 2018-07-09 04:09:58.003).
- [ ] psychic surgery: psychic surgery gives the card under the opponents deck and not ontop (reported by Ultima3007 / 2018-07-09 18:05:31.558).
- [ ] blood moon: blood moon errat not updated, shock lands shouldn't still need to shock and fastlands always come in untapped now (reported by shroudmaster / 2018-07-09 18:44:27.18).
- [ ] Ajani Counselor: I think one of the new M19 Ajanisi is bugged. Text is broken and third ability doesn't work. (reported by maxwarrior / 2018-07-10 02:04:27.773).
- [ ] Bug: Emblem for Ajani, Adversary of Tyrants (Core 2019) isn't producing tokens at end step. (reported by SilverSliver / 2018-07-10 03:56:04.539).
- [ ] M19 Ajani: Emblem does nothing. (reported by Blue / 2018-07-10 04:59:09.695).
- [ ] Vivien Reid: Her -3 functions as a -X, you can -0 atm. (reported by Blue / 2018-07-10 05:00:35.294).
- [ ] Skyrider Patrol Bug: When "Skyrider Patrols" (M19) Ability triggers, you sometimes can choose more than one target. (reported by Sliveroverlord / 2018-07-10 12:39:31.563).
- [ ] Decklist isn't rendering as list: During earlier released you could render decks as lists, now the buttons are missing for that option. Why? (reported by Dragas / 2018-07-10 20:22:36.715).
- [ ] Epochrasite bug: It comes out of suspend, is cast and enters the battlefield. THEN if it's returned to its owner's hand from battlefield, the bounced Epochrasite can't be cast for the rest of the game. (reported by Sickler / 2018-07-10 21:09:32.85).
- [ ] New plainswalker: vivien reid' option '-3'e'tezzeret, artifice master' option '-9' (reported by stingray / 2018-07-10 23:47:53.816).
- [ ] planeswalker emblem bug: emblems that have triggered effects do not trigger for the m19 planeswalkers ie Ajani, advesary of tyrants emblem (reported by No1OConsequens / 2018-07-11 10:12:10.42).
- [ ] DRS: an enemy was just able to cast Death Rite Shaman in Legacy even though it is banned (reported by fekal / 2018-07-11 16:37:55.282).
- [ ] searching library is seemed to be bugged: cant search my library at all. Saccing inventor's fair is seemingly not working (reported by Vyto / 2018-07-11 19:55:55.808).
- [ ] 2x Damping Sphere: When 2 Damping Spheres are in play, the game asks which replacement effect applies at every single time I get priority (assuming there's a land affected) (reported by Bob49 / 2018-07-12 01:34:15.845).
- [ ] Vivien Reid: has a -x loyal ability instead the -3 (reported by LeatherStone49 / 2018-07-12 14:44:53.349).
- [ ] triggers: vaevictus attack trigger hits a poison tip archer, does the poison tip archer that enters the battlefield from vaevictus ability see the original permanents being sacrificed? (reported by BReason / 2018-07-12 15:02:36.787).
- [ ] trigger: poison tip archer potential triggering off it's own death (reported by BReason / 2018-07-12 15:10:18.904).
- [ ] Approach fix: Approach of the second sun has a bug where it tries to make you pick a trigger and ends up not casting it. (reported by General_Bliz / 2018-07-12 20:03:55.972).
- [ ] Heroic Reinforcements: The etb of Heroic Reinforcements gives haste to lands, and maybe other permanents. (reported by fireshoes / 2018-07-13 00:07:58.494).
- [ ] Stone Quarry: Opponent played Stone Quarry. It entered the battlefield untapped for no apparent reason. (reported by fireshoes / 2018-07-13 00:08:57.748).
- [ ] fix vivien reid: 2nd ability says -X but is supposed to be -3 (reported by dsn / 2018-07-13 02:29:29.713).
- [ ] M19 Ajani emblem bug: The emblem doesn't trigger at end of turn (reported by jerjern / 2018-07-13 05:29:12.426).
- [ ] Enlightened Tutor: Seems like playing enlightened tutor crashes me fromthe server and once the card "resolves" without me actually tutoring for something everything goes back to normal (reported by HoodieChris / 2018-07-13 06:18:18.328).
- [ ] m19 dual lands: m19 dual lands are in the basic land slot, not in a common slot (reported by keybee / 2018-07-13 12:17:04.763).
- [ ] Skyrider Patrol: Incremental +1/+1 counters (reported by BadGlitch / 2018-07-13 15:03:59.308).
- [ ] Skyrider Patrol: Triggered twice at beginning of combat, allowing my opponent to put two +1/+1 counters on another creature. (reported by fireshoes / 2018-07-13 17:34:46.972).
- [ ] command anthology 2: can't download commander anthology 2 images (reported by igniteaxiom / 2018-07-13 22:46:22.175).
- [ ] Detection Tower: Leyline of Sanctity isn't allowing me to target them even after using detection tower's ability. (reported by pie / 2018-07-14 02:18:33.363).
- [ ] Mana Flare: Mana flare produces an addition mana (3 in total) when a multicolor land is tapped to produce manaex. sunken hallow and path of ancestry (reported by Erevain / 2018-07-14 04:51:13.178).
- [ ] How do I play against bots ?: when I'm creating a game and adding draftbot as a player it says that computer can't join this table. is this a bug or am I doing something wrong (reported by ThatGinger / 2018-07-14 14:27:18.205).
- [ ] bug on vivien reid: In game, vivien has -x as the 2nd ability. it should be -3 (reported by crowleynv / 2018-07-14 18:39:24.607).
- [ ] sarkhan fireblood: =+2 add two mana of any colors doesn't actually add mana to your mana pool. (reported by vespel / 2018-07-14 23:06:00.314).
- [ ] Warrior Bug: Mindblade Render is not triggering =[ (reported by Sethery1994 / 2018-07-15 01:47:30.739).
- [ ] field of ruin bugged: doesn't let u search for a land. (reported by vespel / 2018-07-15 03:18:09.356).
- [ ] Commander time out bug: During a game of 4 player commander, one player seemingly wentafk. After 300 seconds, it kicked the wrong player instead of theone we were waiting on. (reported by SpellSnipe / 2018-07-15 05:07:51.437).
- [ ] Players leaving the game: It's really annoying when someone leaves the game and doesnt come back. can you put like a 3 minute timer on that and make them auto leave? (reported by grouphug / 2018-07-15 07:40:56.214).
